client/app/views/labels/admin.coffee

```diff
@@ -0,0 +1,72 @@
+{Filtered, Sorted}  = BackboneProjections
+
+module.exports = class LabelsAdminToolView extends Mn.CompositeView
+
+    template: require 'views/templates/labels/admin'
+
+    tagName: 'fieldset'
+
+
+    behaviors:
+        Navigator: {}
+
+
+    childViewContainer: 'ul'
+
+    childView: require 'views/labels/select'
+
+    childEvents:
+        'label:update': 'changeSelection'
+
+
+    modelEvents:
+        'change:selected': 'updateSelection'
+
+
+    initialize: (args) ->
+        super args...
+        @listenTo @, 'change:tags', @updateSelection
+
+
+    select: (model, models) =>
```

Comment:
This one is related to https://github.com/cozy/cozy-ui/issues/14. The CSS of the checkbox will be handled by _cozy-ui_, and view'll only need to update the `aria-checked` state for the concerned checkboxes.